analyses/tp53_nf1_score/02-qc-rna_expression_score.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "QC correlation of classifier score with RNA-Seq expression"
+author: "K S Gaonkar (D3B)"
+output: html_notebook
+
+---
+    
+In this notebook we will find correlation between RNA-Seq and classifier score
+We will check if the values are inversely correlated as we would expect?
+
+### Setup
+```{r}
+library("ggpubr")
+library("ggthemes")
+library("tidyverse")
+library("broom")
+
+# rootdir
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+data_dir <- file.path(root_dir, "data")
+results_dir <- "results"
+```
+
+### Classifier scores
+
+ 1) Read in scores from tp53-nf1-classifier for stranded
+```{r}
+
+score_stranded_df <- read_tsv(file.path(results_dir, "pbta-gene-expression-rsem-fpkm-collapsed.stranded_classifier_scores.tsv")) %>%
+  dplyr::select(
+    "tp53_score",
+    "sample_id"
+  ) %>%
+  unique() %>%
```

Comment:
```suggestion
```
I think this shouldn't be necessary since there should be only one score per BS_ID already, right?